services/SchedulableTransactionBody.proto

```diff
@@ -0,0 +1,116 @@
+    ContractUpdateTransactionBody contractUpdateInstance = 5; // Updates a contract
+    ContractDeleteTransactionBody contractDeleteInstance = 6; //Delete contract and transfer remaining balance into specified account
+
+    CryptoCreateTransactionBody cryptoCreateAccount = 7; // Create a new cryptocurrency account
+    CryptoDeleteTransactionBody cryptoDelete = 8; // Delete a cryptocurrency account (mark as deleted, and transfer hbars out)
+    CryptoTransferTransactionBody cryptoTransfer = 9; // Transfer amount between accounts
+    CryptoUpdateTransactionBody cryptoUpdateAccount = 10; // Modify information such as the expiration date for an account
+
+    FileAppendTransactionBody fileAppend = 11; // Add bytes to the end of the contents of a file
+    FileCreateTransactionBody fileCreate = 12; // Create a new file
+    FileDeleteTransactionBody fileDelete = 13; // Delete a file (remove contents and mark as deleted until it expires)
+    FileUpdateTransactionBody fileUpdate = 14; // Modify information such as the expiration date for a file
+    SystemDeleteTransactionBody systemDelete = 15; // Hedera administrative deletion of a file or smart contract
+    SystemUndeleteTransactionBody systemUndelete = 16; //To undelete an entity deleted by SystemDelete
+    FreezeTransactionBody freeze = 17; // Freeze the nodes
+
+    ConsensusCreateTopicTransactionBody consensusCreateTopic = 18; // Creates a topic
+    ConsensusUpdateTopicTransactionBody consensusUpdateTopic = 19; // Updates a topic
+    ConsensusDeleteTopicTransactionBody consensusDeleteTopic = 20; // Deletes a topic
+    ConsensusSubmitMessageTransactionBody consensusSubmitMessage = 21; // Submits message to a topic
+
+    TokenCreateTransactionBody tokenCreation = 22; // Creates a token instance
+    TokenFreezeAccountTransactionBody tokenFreeze = 23; // Freezes account not to be able to transact with a token
+    TokenUnfreezeAccountTransactionBody tokenUnfreeze = 24; // Unfreezes account for a token
+    TokenGrantKycTransactionBody tokenGrantKyc = 25; // Grants KYC to an account for a token
+    TokenRevokeKycTransactionBody tokenRevokeKyc = 26; // Revokes KYC of an account for a token
+    TokenDeleteTransactionBody tokenDeletion = 27; // Deletes a token instance
+    TokenUpdateTransactionBody tokenUpdate = 28; // Updates a token instance
+    TokenMintTransactionBody tokenMint = 29; // Mints new tokens to a token's treasury account
+    TokenBurnTransactionBody tokenBurn = 30; // Burns tokens from a token's treasury account
+    TokenWipeAccountTransactionBody tokenWipe = 31; // Wipes amount of tokens from an account
+    TokenAssociateTransactionBody tokenAssociate = 32; // Associate tokens to an account
+    TokenDissociateTransactionBody tokenDissociate = 33; // Dissociate tokens from an account
+
+    ScheduleDeleteTransactionBody scheduleDelete = 43; // Marks a schedule in the network's action queue as deleted, preventing it from executing
```

Comment:
If we're not matching the field indexes, then this shouldn't jump from 33 to 43.

Reply:
Fixed.